tabs/tab_survival.py

```
@@ -495,39 +515,54 @@ def out_curves_result():
     def out_curves_table():
-        """Render curves statistics table."""
         res = curves_result.get()
         return render.DataGrid(res['stats']) if res else None
+
+    @render.data_frame
+    def out_surv_times_table():
+        res = curves_result.get()
+        return render.DataGrid(res['surv_at_times']) if res and res.get('surv_at_times') is not None else None
     
     @render.download(filename="survival_report.html")
     def btn_dl_curves():
```

Ruff (0.14.10)

Missing return type annotation for private function `btn_dl_curves`

(ANN202)